lib/secure-boot-lib.robot

```diff
@@ -236,7 +236,64 @@ Check Secure Boot In Windows
     ...    Should Contain    ${out}    True
     RETURN    ${sb_status}
 
-Check Gf Attempt Secure Boot Can Be Selected
+Enable Secure Boot
```

Comment:
I'm not sure if we need this kwd. Wy try to favor smaller steps in keywords, so you can get a good idea on the test scenario just by reading the test case (which will be generated as a documentation), so it can also be reproduced manually.   It can be implemented in just 3 lines with this MR: https://github.com/Dasharo/open-source-firmware-validation/pull/95

If we will use it often, then let's add it, but as below.
```
    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
    Set Option State    ${sb_menu}    Enable Secure Boot    ${TRUE}
    Save Changes And Reset    2
```

Reply:
Or maybe we do need them, I'm not sure at this point...
https://github.com/Dasharo/open-source-firmware-validation/pull/95/files#diff-50de80694206570830c0d9e0dd8bc964352cf44db51a4c3eab6368dab4dea03fR123

Reply:
@macpijan I feel we wasted some energy on this coordination because you seem to face the same issues I faced and tried to resolve. Of course, my solution may not be up to the expected quality, but that should be the point of your review. I'm unsure how to resolve those experiments and ensure we inform each other about vision, plans, and achievements so the other side will not repeat the same mistakes and will not be affected by releases, API changes, or different visions of implementation.

Please note `Select Enable Secure Boot Option` use ` Check If Enable Secure Boot Can Be Selected ` to handle precisely your concern from the linked TODO comment. Is it a good design? That is the question.

Reply:
@pietrushnic Please just use my MR: https://github.com/Dasharo/open-source-firmware-validation/pull/95#issuecomment-1805513832

Reply:
@macpijan, please let me know next time that it doesn't make sense to spend time when you plan to solve stuff yourself.